migrations/808-waffle-automatic-validation.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO waffle_flag (name, everyone, percent, testing, superusers, staff, authenticated, languages, rollout, note, created, modified)
+       VALUES ('automatic-validation', 0, NULL, 0, 0, 0, 0, '', 0, "Guards the automatic validation/review/signing or unlisted add-ons when they're submitted.", NOW(), NOW());
```

Comment:
This doesn't seem like something you'd "roll out" slowly to a subset of users, so it seems better suited to a waffle switch. Or do you plan to add select users to this flag at first?